src/plugins/discover/public/application/components/sidebar/discover_sidebar.scss

```diff
@@ -27,7 +48,8 @@
   .euiButtonEmpty__content {
     font-size: $euiFontSizeXS;
     font-weight: $euiFontWeightSemiBold;
-    justify-content: flex-end;
+    justify-content: flex-start;
+    padding-left: $euiSizeM + 2px;
```

Comment:
why 2px?

Reply:
The design had a 2px spacing between the section headers and the elements below them.